docs/pages/enroll-resources/workload-identity/getting-started.mdx

```diff
@@ -23,125 +24,142 @@ You can then connect your workloads to this endpoint to receive SPIFFE SVIDs.
   workloads which need to access Teleport Workload Identity will run. For more
   information, see the [deployment guides](../machine-id/deployment/deployment.mdx).
 
-## Step 1/4. Configure RBAC
+## Step 1/4. Configure Workload Identity
 
-First, Teleport must be configured to allow the Bot to issue SPIFFE SVIDs.
-This is done by configuring a role to allow the issuance of SPIFFE SVIDs and
-then granting this role to the Bot.
+First, you will need to create a WorkloadIdentity resource.
```

Comment:
I would use "Workload Identity" for the resource name so we can use it regardless of Teleport API client, e.g., `tctl` the Kubernetes Operator, etc. In `tctl`, for example, we don't camelcase resource names.

Reply:
[b67f3e0](https://github.com/gravitational/teleport/pull/51501/commits/b67f3e013d69c51cb97e33ffed6fa42b44a6465e)